src/server/script_mgr.cc

```diff
@@ -150,6 +150,10 @@ void ScriptMgr::ConfigCmd(CmdArgList args, ConnectionContext* cntx) {
 
   UpdateScriptCaches(key, data);
 
+  // Schedule empty callback inorder to journal command via transaction framework.
+  cntx->transaction->EnableShards(0);
+  cntx->transaction->ScheduleSingleHop([&](auto* t, auto* shard) { return OpStatus::OK; });
```

Comment:
1. I agree EnableShards(0) looks confusing as if its the number 🤔 
2. I added the hop to `SCRIPT CONFIG` as well because it should also be replicated

Reply:
I think its confusing that EnableShards is called here and not in Init of transaction.
We should call it if the flag NO_KEY_TRANSACTIONAL is set

Reply:
I see you need somehow to differentiate if the call should run on all shards or a single shard, maybe use different flags.
I just think that we may forget calling EnableShards in some cases, calling it in init of transaction makes this more safe